analysis/filter/MergeFilter/README.md

```diff
@@ -13,9 +15,16 @@ specific strategy. Currently there are two main strategies:
 - recursive (`--recursive`) (default): leave structure of additional files. This will also merge optional edges.
 - leaf (`--leaf`) (beta):  fit leaf nodes into reference structure according to their name (and tail of their path), 
 either adding missing leaves (`--add-missing`) or ignoring them (default)
+- leaf (`--leaf`) (beta):  fit leaf nodes into reference structure according to their name (and tail of their path)
```

Comment:
This line is duplicated isn't it?

Reply:
it most certainly is, i don't know what my intention was here. removed it.